ivadomed/transforms.py

```diff
@@ -287,6 +294,50 @@ def __call__(self, sample, metadata=None):
         return data_out, metadata
 
 
+class VertebralSplitting(ImedTransform):
+    """Create an intervertebral disk masks for pose estimation model .
+    NB: This transform is not multichannel capable. It is meant to be used
+    to *create* a multichannel image from a single-channel image.
+    """
+
+    def __init__(self, max_joint=11):
+        self.max_joint = max_joint
+
+    def undo_transform(self, sample, metadata=None):
+        return sample, metadata
+
+    def get_posedata(self, msk, num_ch):
+        ys = msk.shape
+        ys_ch = [np.zeros([ys[0], ys[1]])] * num_ch
+        msk_uint = np.uint8(np.where(msk>0.2, 1, 0))
+        
+        labels_im, num_labels = measure.label(msk_uint, background=0, return_num=True)
```

Comment:
I've removed the rotation code. Now, when I open up the preprocessed images in FSLeyes, they appear properly oriented in the S-I direction.

![Screenshot from 2021-07-09 10-35-19](https://user-images.githubusercontent.com/16181459/125094586-642d9f00-e0a1-11eb-9bec-f62b9757b9c5.png)

When loading these images during training, the resulting numpy array seems to be upside-down.

![tmpm5yhyxia](https://user-images.githubusercontent.com/16181459/125094820-9212e380-e0a1-11eb-8504-6763b5540fd6.PNG)

Is this because numpy arrays have a [top-left origin](https://stackoverflow.com/questions/51849547/why-is-numpys-origin-at-the-upper-left-unlike-mathematical-convention/51849585), while MRI convention is to use a bottom-left origin? (My apologies if this is a really basic question.)

Reply:
> I notice when loading these images during training, the resulting numpy array seems to be upside-down.

are you sure this is the corresponding image? The number of points differ (8 vs. 9) and the pattern seem slightly different. But anyways, I think the key is to have the MRI image and the ground truth (ie: heatmap) treated the same way, and it won't import how the matrix is ordered with numpy

Reply:
> are you sure this is the corresponding image? The number of points differ (8 vs. 9) and the pattern seem slightly different.

It's not, sorry -- I just grabbed a random sample image to demonstrate the orientation.

> But anyways, I think the key is to have the MRI image and the ground truth (ie: heatmap) treated the same way, and it won't import how the matrix is ordered with numpy

:+1: